features/support/env.rb

```diff
@@ -21,6 +21,9 @@
 # Set up basic URLs
 Capybara.app_host = ENV["GOVUK_WEBSITE_ROOT"]
 
+# Set up optional error reporting (using SENTRY_CURRENT_ENV for the environment)
```

Comment:
I'd drop the word 'optional' as it implies it can be configured per-test, whereas really we're opting the entire test suite into Sentry error reporting. (It may be optional whether we do it or not, but as we've decided to do it, I don't think there's value in this word 😁 )

```suggestion
# Set up error reporting (using SENTRY_CURRENT_ENV for the environment)
```

Reply:
Ah yes I see what you mean.

Removed in https://github.com/alphagov/smokey/pull/931/commits/2c950649bb8a769beb917eee2b360ba66efbb99e.